src/workflows/file-resolvers.ts

```diff
@@ -2,19 +2,23 @@ import findParentDir from "find-parent-dir";
 import fs from "fs-extra";
 import path from "path";
 import { OpenAPIV3 } from "@useoptic/openapi-utilities";
-import { LogUpdatingSpecification } from "./logs";
-import chalk from "chalk";
+import { loadVervetResoucePaths } from "./vervet-resolver";
 
 export async function resolveResourcesDirectory(
   workingDirectory: string = getSweaterCombWorkingDirectory(),
```

Comment:
@cmars do we still need this or do we always want to assume it's in PWD? Will it be set via the env to be the project root somehow?

Reply:
I think I'll leave it for now. It might not be set, in which case the default is "use the first resources directory in the first API". Setting it allows one to:
* Select a different API / resources directory within the same project
* Operate on a project elsewhere, when your CWD isn't in the project